File: FJT.ReportViewer/MySqlDBModel/CompanyInfo.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace FJT.ReportViewer.MySqlDBModel
{
    [Table("company_info")]
    public class CompanyInfo
    {
        [Key]
        public int id { get; set; }

        [StringLength(255)]
        public string name { get; set; }

        [StringLength(255)]
        public string personName { get; set; }

        [StringLength(255)]
        public string registeredEmail { get; set; }

        [StringLength(5)]
        public string contactCountryCode { get; set; }

        [StringLength(20)]
        public string contactNumber { get; set; }

        [StringLength(5)]
        public string faxCountryCode { get; set; }

        [StringLength(20)]
        public string faxNumber { get; set; }

        [StringLength(8)]
        public string phoneExt { get; set; }

        [StringLength(255)]
        public string street1 { get; set; }

        [StringLength(255)]
        public string street2 { get; set; }

        [StringLength(255)]
        public string street3 { get; set; }

        [StringLength(255)]
        public string city { get; set; }

        [StringLength(255)]
        public string state { get; set; }

        [StringLength(255)]
        public string postalCode { get; set; }

        [StringLength(255)]
        public string companyLogo { get; set; }

        [StringLength(100)]
        public string ein { get; set; }

        public string remittanceAddress { get; set; }

        public int mfgCodeId { get; set; } // foreign key

        public int countryID { get; set; } // foreign key

        public bool? isDeleted { get; set; }

        [StringLength(255)]
        public string createdBy { get; set; }

        [StringLength(255)]
        public string updatedBy { get; set; }

        [StringLength(255)]
        public string deletedBy { get; set; }

        public DateTime createdAt { get; set; }

        public DateTime updatedAt { get; set; }

        public DateTime? deletedAt { get; set; }

        public int? createByRoleId { get; set; }

        public int? updateByRoleId { get; set; }

        public int? deleteByRoleId { get; set; }

        [ForeignKey("mfgCodeId")]
        public virtual Mfgcodemst Mfgcodemst { get; set; }
    }
}

File: fjt.api/models/CompanyInfo.js
const Sequelize = require('sequelize');

module.exports = (sequelize) => {
    const CompanyInfo = sequelize.define('CompanyInfo', {
        id: {
            type: Sequelize.INTEGER,
            allowNull: false,
            primaryKey: true
        },
        name: {
            type: Sequelize.STRING,
            allowNull: false
        },
        personName: {
            type: Sequelize.STRING,
            allowNull: true
        },
        registeredEmail: {
            type: Sequelize.STRING,
            allowNull: false
        },
        contactCountryCode: {
            type: Sequelize.STRING,
            allowNull: true
        },
        contactNumber: {
            type: Sequelize.STRING,
            allowNull: true
        },
        faxCountryCode: {
            type: Sequelize.STRING,
            allowNull: true
        },
        faxNumber: {
            type: Sequelize.STRING,
            allowNull: true
        },
        phoneExt: {
            type: Sequelize.INTEGER,
            allowNull: true
        },
        street1: {
            type: Sequelize.STRING,
            allowNull: true
        },
        street2: {
            type: Sequelize.STRING,
            allowNull: true
        },
        street3: {
            type: Sequelize.STRING,
            allowNull: true
        },
        city: {
            type: Sequelize.STRING,
            allowNull: true
        },
        state: {
            type: Sequelize.STRING,
            allowNull: true
        },
        postalCode: {
            type: Sequelize.STRING,
            allowNull: true
        },
        isDeleted: {
            type: Sequelize.BOOLEAN,
            allowNull: true
        },
        createdBy: {
            type: Sequelize.STRING,
            allowNull: true
        },
        mfgCodeId: {
            type: Sequelize.INTEGER,
            allowNull: true
        },
        countryID: {
            type: Sequelize.INTEGER,
            allowNull: true
        },
        updatedBy: {
            type: Sequelize.STRING,
            allowNull: true
        },
        unitOfTime: {
            type: Sequelize.STRING,
            allowNull: true
        },
        companyLogo: {
            type: Sequelize.STRING,
            allowNull: true
        },
        createByRoleId: {
            type: Sequelize.INTEGER,
            allowNull: true
        },
        updateByRoleId: {
            type: Sequelize.INTEGER,
            allowNull: true
        },
        deleteByRoleId: {
            type: Sequelize.INTEGER,
            allowNull: true
        },
        ein: {
            type: Sequelize.STRING,
            allowNull: true
        },
        remittanceAddress: {
            type: Sequelize.TEXT,
            allowNull: true
        },
        legalName: {
            type: Sequelize.STRING,
            allowNull: true
        }
    },
        {
            paranoid: true,
            tableName: 'company_info'
        });

    CompanyInfo.associate = (models) => {
        CompanyInfo.belongsTo(models.CountryMst, {
            as: 'countryMst',
            foreignKey: 'countryID'
        });
        CompanyInfo.belongsTo(models.MfgCodeMst, {
            as: 'MfgCodeMst',
            foreignKey: 'mfgCodeId'
        });
    };

    return CompanyInfo;
};